src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -363,6 +364,18 @@ public MethodInfo(AssemblyInfo assembly, MethodDefinitionHandle methodDefHandle,
 
                 StartLocation = new SourceLocation(this, start);
                 EndLocation = new SourceLocation(this, end);
+
+                foreach (var cattr in methodDef.GetCustomAttributes())
+                {
+                    var ctorHandle = asmMetadataReader.GetCustomAttribute(cattr).Constructor;
+                    if (ctorHandle.Kind == HandleKind.MemberReference)
+                    {
+                        var container = asmMetadataReader.GetMemberReference((MemberReferenceHandle)ctorHandle).Parent;
+                        var name = asmMetadataReader.GetString(asmMetadataReader.GetTypeReference((TypeReferenceHandle)container).Name);
+                        if (name == "DebuggerHiddenAttribute")
+                            this.IsHiddenFromDebugger = true;
```

Comment:
is it missing break?

Reply:
Resolved in [PR](https://github.com/dotnet/runtime/pull/61165).